user-guide/dataminer_services/Catalog/Change_email_dataminer_services_account.md

```diff
@@ -5,21 +5,69 @@ keywords: cloud email, cloud account
 
-   This will create a new account linked to the new email address.
+### Linking to your DataMiner account
 
-1. Visit [community.dataminer.services](https://community.dataminer.services/) with the new account.
+If your dataminer.services account was previously linked to your DataMiner account, you will need to create this link again.
 
-1. Contact [back.office@skyline.be](mailto:back.office@skyline.be?subject=Request%20to%20transfer%20my%20DataMiner%20Dojo%20data&body=Hi%2C%0D%0A%0D%0AI%20would%20like%20to%20transfer%20my%20DataMiner%20Dojo%20data.%0D%0AThe%20accounts%20involved%20are%20listed%20below%3A%0D%0A%0D%0AFrom%3A%0D%0A%3Cfill%20in%20email%20address%3E%0D%0A%0D%0ATo%20(new%20account)%3A%0D%0A%3Cfill%20in%20email%20address%3E%0D%0A) to have your existing data transferred to the new account.
+For information on how to do so, see [Linking your DataMiner account to your dataminer.services account](xref:Linking_your_DataMiner_and_dataminer_services_account).
+
+### Regaining access to projects via the Collaboration app
+
+To maintain access to your projects and tasks via the Collaboration app, your new dataminer.services account will need to be linked to your verified organization. This is something that will need to be done by Skyline staff upon request.
+
+To send this request, after you have logged onto [collaboration.dataminer.services](https://collaboration.dataminer.services/) with your new account, contact [back.office@skyline.be](mailto:back.office@skyline.be) and ask them to link your new account to your verified organization.
+
+> [!TIP]
+>
+> - For more info about the Collaboration app, refer to [About the Collaboration app](xref:About_the_Collaboration_app).
+> - Direct link to Collaboration app: [collaboration.dataminer.services](https://collaboration.dataminer.services/)
+> - See also: [Logging on to dataminer.services > Logging on](xref:Logging_on_to_dataminer_services#logging-on)
+
+### Transferring data on DataMiner Community (Dojo)
+
+Upon logging into Dojo with your new account, you will notice that a new account has been created. If you want to transfer the data from your old account, you will need to contact Skyline. However, note that if you opt for the transfer, your old account will be deleted.
+
+If you wish to transfer your data, after you have logged onto [community.dataminer.services](https://community.dataminer.services/) with your new account, contact [back.office@skyline.be](mailto:back.office@skyline.be?subject=Request%20to%20transfer%20my%20DataMiner%20Dojo%20data&body=Hi%2C%0D%0A%0D%0AI%20would%20like%20to%20transfer%20my%20DataMiner%20Dojo%20data.%0D%0AThe%20accounts%20involved%20are%20listed%20below%3A%0D%0A%0D%0AFrom%3A%0D%0A%3Cfill%20in%20email%20address%3E%0D%0A%0D%0ATo%20(new%20account)%3A%0D%0A%3Cfill%20in%20email%20address%3E%0D%0A) to request to transfer your existing data. However, because some limitations apply (as detailed below), you will need to confirm before the transfer goes ahead.
```

Comment:
```suggestion
If you wish to transfer your data, after you have logged into [community.dataminer.services](https://community.dataminer.services/) with your new account, contact [back.office@skyline.be](mailto:back.office@skyline.be?subject=Request%20to%20transfer%20my%20DataMiner%20Dojo%20data&body=Hi%2C%0D%0A%0D%0AI%20would%20like%20to%20transfer%20my%20DataMiner%20Dojo%20data.%0D%0AThe%20accounts%20involved%20are%20listed%20below%3A%0D%0A%0D%0AFrom%3A%0D%0A%3Cfill%20in%20email%20address%3E%0D%0A%0D%0ATo%20(new%20account)%3A%0D%0A%3Cfill%20in%20email%20address%3E%0D%0A) to request to transfer your existing data. However, because some limitations apply (as detailed below), you will need to confirm before the transfer goes ahead.
```